get/index.tpl.html

```diff
@@ -58,11 +47,14 @@ <h1>Feeling adventurous?</h1>
 		</article>
 	</div>
 
-	<meta property="filename" content="mavo[if(transpile, '.' & transpile)][if(minify, '.' & min)].js">
+	<meta property="filename" content="mavo[if(minify, '.' & min)].js">
 
 	<p>Now copy and paste the following code in your page’s <code>&lt;head></code> section, or click on the links below to download these files locally:</p>
 	<pre class="language-foo"><code>&lt;script src="<a href="https://get.mavo.io/[path][filename]" download>https://get.mavo.io/[path][filename]</a>">&lt;/script>
 &lt;link rel="stylesheet" href="<a href="https://get.mavo.io/[path]mavo[if(minify, '.' & min)].css" download>https://get.mavo.io/[path]mavo[if(minify, '.' & min)].css</a>"></code></pre>
+
+	<p>Looking for another option? You have the choice to <a href="https://www.npmjs.com/package/mavo">install Mavo via npm</a>:</p>
+	<pre class="language-foo"><code>npm install mavo</code></pre>
```

Comment:
No, this should be an option. If they select it, they get appropriate instructions. e.g. `npm install mavo@version` etc